tests/utils/SparseGridTest.cpp

```diff
@@ -43,25 +43,38 @@ TEST_P(GetNearbyTest, GetNearby)
 
-    //Are all near points reported as near?
+    // Are all near points reported as near?
     for (const Point point : parameters.expected_near)
     {
-        EXPECT_NE(result.end(), std::find_if(result.begin(), result.end(), [&point](const typename SparsePointGridInclusive<Point>::Elem &elem) { return elem.val == point; }))
-            << "Point " << point << " is near " << target << " (distance " << vSize(point - target) << "), but getNearby didn't find it. Grid size: " << grid_size;
+        EXPECT_NE(result.end(),
+                  std::find_if(result.begin(),
+                               result.end(),
+                               [&point](const typename SparsePointGridInclusive<Point>::Elem& elem) { return elem.val == point; }))
+          << "Point " << point << " is near " << target << " (distance " << vSize(point - target)
+          << "), but getNearby didn't find it. Grid size: " << grid_size;
     }
-    //Are all far points NOT reported as near?
+    // Are all far points NOT reported as near?
     for (const Point point : parameters.expected_far)
     {
-        EXPECT_EQ(result.end(), std::find_if(result.begin(), result.end(), [&point](const typename SparsePointGridInclusive<Point>::Elem &elem) { return elem.val == point; }))
-            << "Point " << point << " is far from " << target << " (distance " << vSize(point - target) << "), but getNearby thought it was near. Grid size: " << grid_size;
+        EXPECT_EQ(result.end(),
+                  std::find_if(result.begin(),
+                               result.end(),
+                               [&point](const typename SparsePointGridInclusive<Point>::Elem& elem) { return elem.val == point; }))
+          << "Point " << point << " is far from " << target << " (distance " << vSize(point - target)
+          << "), but getNearby thought it was near. Grid size: " << grid_size;
     }
 }
 
-INSTANTIATE_TEST_CASE_P(GetNearbyInstantiation, GetNearbyTest, testing::Values(
-    GetNearbyParameters({ Point(0,   100) }, std::unordered_set<Point>(), std::unordered_set<Point>({ Point(0,   100) })), //A far point.
-    GetNearbyParameters({ Point(95,  100) }, std::unordered_set<Point>({ Point(95,  100) }), std::unordered_set<Point>()), //A near point.
-    GetNearbyParameters({ Point(100, 100) }, std::unordered_set<Point>({ Point(100, 100) }), std::unordered_set<Point>())  //On top of the target.
-));
+INSTANTIATE_TEST_CASE_P(
+  GetNearbyInstantiation,
+  GetNearbyTest,
+  testing::Values(
+    GetNearbyParameters({ Point(0, 100) }, std::unordered_set<Point>(), std::unordered_set<Point>({ Point(0, 100) })), // A far point.
+    GetNearbyParameters({ Point(95, 100) }, std::unordered_set<Point>({ Point(95, 100) }), std::unordered_set<Point>()), // A near point.
+    GetNearbyParameters({ Point(100, 100) },
+                        std::unordered_set<Point>({ Point(100, 100) }),
+                        std::unordered_set<Point>()) // On top of the target.
```

Comment:
Why is this last line broken up but not the other ones? It rather breaks the readability of having a separate test case on each line.